src/containers/Layout/NotificationDrawerWrapper.tsx

```diff
@@ -1,42 +1,70 @@
 import {
+  Button,
   NotificationDrawer,
   NotificationDrawerBody,
   NotificationDrawerHeader,
   NotificationDrawerList,
   NotificationDrawerListItem,
   NotificationDrawerListItemBody,
-  NotificationDrawerListItemHeader
+  NotificationDrawerListItemHeader,
+  ButtonVariant
 } from '@patternfly/react-core';
-import React, { useContext } from 'react';
+import React, { useContext, useEffect, useState } from "react";
 import { AppContext } from '../../context/context';
+import { TimesIcon, TimesCircleIcon } from "@patternfly/react-icons";
+import { NotificationActionTypes } from "../../context/actions/types";
 
 interface NotificationDrawerWrapperProps {
   onClose: () => void;
 }
 
 const NotificationDrawerWrapper: React.FC<NotificationDrawerWrapperProps> = ({ onClose }) => {
-  const { state: { notifications } } = useContext(AppContext);
+  const [isClearAllDisabled, setIsClearAllDisabled] = useState(true);
```

Comment:
Would it be better to just check the length of notifications to determine whether or not to disable the Clear all button?